pages/starters/nextjs.mdx

```diff
@@ -1,6 +1,5 @@
 ---
 title: "Unbody Next.js Starter"
-authors: [{name: "Amir Houieh", link: "https://www.linkedin.com/in/amirhouieh/", image: "amir.png"}]
```

Comment:
Is this needed? Where is it being used? The json is causing issues when exporting.